tools/check_requirements/README.md

```diff
@@ -26,23 +26,36 @@ Tested on :
 
 For Mac OSX and Linux :
 
-- `_config.sh` : file containing the configuration of checker tool (tools, versions, debug)
+- `_config.sh` : file for loading configuration parameters
```

Comment:
please remove this line for documentation, deprecated with last upgrade

Reply:
done